### PR TITLE
Feature/obg 13 bank search api

### PR DESCRIPTION
**Pull request by max402**

### CODE REVIEW

File: core/banking-protocol/pom.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            <groupId>org.skyscreamer</groupId>
+            <artifactId>jsonassert</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- BouncyCastle to generate P12 keystores -->
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>1.64</version>
```
Comment (electronicpeter) [inline core/banking-protocol/pom.xml]: maybe no hard coded version numbers -> properties to main pom.xml

File: core/banking-protocol/pom.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>1.12.1</version>
```
Comment (electronicpeter) [inline core/banking-protocol/pom.xml]: maybe no hard coded version numbers -> properties to main pom.xml

File: pom.xml [hunk 3]
```diff
@@ -170,21 +175,30 @@
                 <version>${fasterxml-data-format.version}</version>
             </dependency>
 
-            <!-- jupiter -->
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
+            </dependency>
```
Comment (electronicpeter) [inline pom.xml]: maybe scope should be test

File: pom.xml [hunk 4]
```diff
@@ -170,21 +175,30 @@
                 <version>${fasterxml-data-format.version}</version>
             </dependency>
 
-            <!-- jupiter -->
+            <dependency>
```
Comment (electronicpeter) [inline pom.xml]: guava is not jupiter -> wrong comment

Comment (max402) [reply pom.xml]: sure. it's mess with pom.xml after few merges

**Comment by electronicpeter** [comment]
I am afraid it was my fault..., sorry.

File: core/banking-protocol/pom.xml (hunk 1 above)
Comment (max402) [reply core/banking-protocol/pom.xml]: ok

File: core/banking-protocol/pom.xml (hunk 2 above)
Comment (max402) [reply core/banking-protocol/pom.xml]: ok

**Pull request merged**